### PR TITLE
Autogenerated new docs and demo at Fri May 5 2017 17:09:14

### PR DESCRIPTION
**Pull request by ilyavolodin**

### CODE REVIEW

**Comment by not-an-aardvark** [comment]
...never mind, too late.

File: docs/4.0.0/user-guide/migrating-to-4.0.0.md [hunk 1]
```diff
@@ -77,8 +77,6 @@ To make the upgrade process easier, we've introduced the [`indent-legacy`](/docs
 
 ## <a name="config-validation"/> Unrecognized properties in config files now cause a fatal error
 
-**Note:** This feature is a work in progress and has not been released yet.
```
Comment (platinumazure) [inline docs/4.0.0/user-guide/migrating-to-4.0.0.md]: Should this line be preserved?

Comment (not-an-aardvark) [inline docs/4.0.0/user-guide/migrating-to-4.0.0.md]: No, it's been released now. I added that line intentionally as part of https://github.com/eslint/eslint/commit/734846b1557a45eb43286c4da2bdc1e13b89d1c6.

Comment (ilyavolodin) [inline docs/4.0.0/user-guide/migrating-to-4.0.0.md]: Good point. We should fix it.

Comment (ilyavolodin) [inline docs/4.0.0/user-guide/migrating-to-4.0.0.md]: Wait, sorry. We did merge this in, so this is correctly removed.

Comment (platinumazure) [inline docs/4.0.0/user-guide/migrating-to-4.0.0.md]: Thanks for clarifying @not-an-aardvark, I agree no change is needed.

**Pull request merged**